### ISSUE TITLE
[COMPONENT] Action Menu

### ISSUE THREAD

**Issue by mictro** [kirbydesign/designsystem#2807]
## Describe the solution you'd like
This issue describes the revised Action Menu component as specified in Sketch - https://www.sketch.com/s/af90babd-c91f-4870-b097-ce26b590ed59/p/13C1424F-FEC5-4F70-B34B-026A0A4585AE/canvas    
At the time of writing, the design guidelines for the new component are not present in zeplin. 

They general UX guidelines for the Action Sheet reads as follows:
_Action sheets are used as an overflow menu that appears in response to a control or action. An action sheet displays a set of two or more choices related to the current context. It appears on top of the app's content, and must be manually dismissed by the user before they can resume other interaction._

The Action Sheet look and work the same on the web and app.
There are two types of Action Sheets:
 	- Simple: Only actions
	- Header + subheader: Title, subtitle (optional), and actions.

The Action Sheet acts almost like a dropdown. It appears below the control that triggered the action sheet and is without a background layer. 

The Action Sheet is to be opened in response to a button action which is not part of the component. The Action Sheet should appear as a popover close to the button at the 'best possible' location with respect to the available space (not clarified yet - ask UX).
The popover functionality is to be provided by a new Kirby component built by a developer under the corporation contribution regime (at the time of writing unavailable).     
The Action Sheet should close when an action is triggered, escape is pressed or the background layer is clicked.

<img width="1046" alt="Skærmbillede 2023-01-25 kl  12 50 47" src="https://user-images.githubusercontent.com/18654594/214562567-045bcdad-ce8e-41c9-be2b-ebf5bbdd7e15.png">

<img width="171" alt="Skærmbillede 2023-01-25 kl  13 08 45" src="https://user-images.githubusercontent.com/18654594/214559780-b2de2fd0-f326-4ee4-9e1e-3eb299ad96f3.png">

## Acceptance criterias
- [x] The sheet appears as a popover both in normal and modal context.
- [x] The sheet popover will dynamically adjust it's position in order to appear at the optimal place. (align with UX)
- [x] The user is able to dismiss the sheet by pressing escape or tapping/clicking the background.
- [x] The user is able to trigger an action notifying registered listeners.
- [x] The sheet closes when an action is triggered.
- [x] The sheet conforms to the design put forward in sketch/zeplin.

Initial Estimate: 13

## Are there any additional context?
No
<hr />

### Clarifications 
During the estimation session the following questions were posed: 
- [x] Max height and scroll bar?
   ja, sættes af bruger - scrollbar.
- [ ] Icon in header?
   Nej, måske senere.
- [ ] Animation on open?
   Nej
- [ ] Enforced location?
  Overvejes
- [ ] Action sheet - right name?
    Finder action-overlay nyt navn men bevarer design.  
- [ ] Reuse dropdown?
   No, new component.

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-refinement); do not proceed until this is done.
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#tech-refinement); do not proceed until this is done.

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by NillerW** [comment]
Related to #2744

**Comment by NillerW** [comment]
Completed by #2744
